docs/v4/index.md

```diff
@@ -22,4 +22,5 @@ api
 nojit
 TODO
 Parcels v4 Project Board <https://github.com/orgs/OceanParcels/projects/5>
+Parcels v4 migration guide <../community/v4-migration>
```

Comment:
Linking it as well from this page, but once v4 is released this whole "v4" section will be deleted (we probably might want to do other restructuring as well, to be decided/discussed later)